integration/benchmark/follower_test.go

```diff
@@ -52,9 +52,12 @@ func TestTxFollower(t *testing.T) {
 	require.Equal(t, blockID, f.BlockID())
 
 	// test that transactionID is eventually discovered
-	unittest.AssertClosesBefore(t, f.Follow(transactionID), 1*time.Second)
-	// test that random transaction is not found
-	unittest.AssertNotClosesBefore(t, f.Follow(flowsdk.Identifier{0x7}), 200*time.Millisecond)
+	unittest.AssertReturnsBefore(t, func() { <-f.Follow(transactionID) }, 1*time.Second)
+	select {
+	case <-f.Follow(flowsdk.Identifier{0x7}):
```

Comment:
can a comment be added here that explains why the random `0x7` value was put here? (assuming that it tests that a random transaction shouldn't be found but can be more explicit)

Reply:
Why can't we use `unittest.AssertNotClosesBefore()` for asserting random transaction not found, just like before?

Reply:
Sadly, `AssertNotClosesBefore` relies on channel being `struct{}` and we have a result there now.